tests/integration/model_contract/models/expected/expected_model_with_invalid_name.sql

```diff
@@ -0,0 +1,7 @@
+{{ config(materialized='table') }}
```

Comment:
we should have a small test with materialization='view' and another small test with file_format='parquet'